packages/astro/test/preview-routing.test.js

```diff
@@ -12,6 +12,7 @@ describe('Preview Routing', () => {
 			before(async () => {
 				fixture = await loadFixture({
 					projectRoot: './fixtures/with-subpath-no-trailing-slash/',
+					dist: new URL('./fixtures/with-subpath-no-trailing-slash/dist-4000/', import.meta.url),
```

Comment:
A third fix needed: The permissions issue wasn't actually fixed 🤦 But, it turns out that `astro-jsx` already solved this issue by running each build to a different `dist` output folder, guaranteeing that the tests down step on each other. We bring in that same approach here.